lib/initializer.rb

```diff
@@ -0,0 +1,16 @@
+class Initializer
+
+  attr_accessor :input, :output, :board
+
+  def initialize(input=STDIN, output=STDOUT)
+    @input = input
+    @output = output
+    @board = Board.new(get_board_size)
```

Comment:
Why is this being called in the constructor?

Reply:
I thought I would call get_board_size inside the Board constructor because get_board_size is the function that will prompt for and save the user's input for the size of the board. 

I refactored so that "get_board_size" (1) uses the display class' public method "get_board_size" to get the user's input and (2) also performs the error handling for the board size. 

I then added another function "create_board" that calls "get_board_size" and passes the board size number into the Board class constructor.

Reply:
@jayden, are you asking why isn't board being passed in to initializer? 

This needs to still be addressed (#3).

Reply:
I am not passing in the Board class into initializer because I need the player's input for the board size to initialize the Board.